frontend/ui/network/manager.lua

```diff
@@ -6,6 +6,7 @@ local LuaSettings = require("luasettings")
 local UIManager = require("ui/uimanager")
 local _ = require("gettext")
 local T = require("ffi/util").template
+local FFIUtil = require("ffi/util")
```

Comment:
Please change these lines to read as follows:
Please observe the coding style guide:

```lualocal ConfirmBox = require("ui/widget/confirmbox")
local DataStorage = require("datastorage")
local Device = require("device")
local FFIUtil = require("ffi/util")
local InfoMessage = require("ui/widget/infomessage")
local LuaSettings = require("luasettings")
local UIManager = require("ui/uimanager")
local _ = require("gettext")
local T = FFIUtil.template
```

That being said, I wonder if it would be more appropriate as `ffiutil` in concordance with `util`, seeing how it's not really a module; pinging @poire-z

Reply:
I had naturally set in the frontend/ui/wikipedia.lua i'm working on:
```lua
local JSON = require("json")
local Screen = require("device").screen
local ffiutil = require("ffi/util")
local logger = require("logger")
local util = require("util")
local _ = require("gettext")
local T = require("ffi/util").template
````

Reply:
ok.